packages/maskbook/src/utils/utils.ts

```diff
@@ -200,7 +200,7 @@ export function assert(x: any, ...args: any): asserts x {
 }
 
 export function checkInputLengthExceed(name: string) {
-    return Array.from(name).length >= WALLET_OR_PERSONA_NAME_MAX_LEN
+    return name.length >= WALLET_OR_PERSONA_NAME_MAX_LEN
```

Comment:
```
'💊'.length
2
Array.from('💊').length
1
```

Reply:
I haven't learned deeply about emoji (An emoji is a glyph which may contains multiple unicode scalars), but I think we can follow twitter and browser's behavior.

![image](https://user-images.githubusercontent.com/1141198/123500278-8b32ac80-d66f-11eb-94e1-09ea7923fa59.png)

![image](https://user-images.githubusercontent.com/1141198/123500284-938ae780-d66f-11eb-9759-29edbc03e55e.png)

https://codesandbox.io/s/jolly-tdd-rqn55?file=/src/App.js